### ISSUE TITLE
Does it support generating ES6 specification JS files and using them in grPC-Web

### ISSUE THREAD

**Issue by yl-yue** [stephenh/ts-proto#568]
Does it support generating ES6 specification JS files and using them in grPC-Web

**Comment by stephenh** [comment]
No, we only generate TypeScript. Technically you could generate TypeScript, then compile that to ES6 output, and it'd work. Or find some other way to strip the types.

**Comment by yl-yue** [comment]
Thank you for your reply.

Can I understand that I can develop in grpc-web using pure TypeScript and therefore there is no JS file ES6 import problem because there are no (or no need for) JS files

**Comment by stephenh** [comment]
I'm not entirely following, but yes you can use pure TypeScript, albeit at some point it'll become JS when you run it. Going to close this out since it's not an issue.
